src/components/Tabs/Tab.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import { getAriaLabel, getTabsClass } from './utils';
+
+const Tab = (props) => {
+  const {
+    children,
+    className,
+    id,
+    isActive,
+    scoped,
+    renderTitle: _,
+    title: __,
+    ...rest
+  } = props;
+
+  return (
+    <div
+      {...rest}
+      aria-labelledby={getAriaLabel(id)}
+      className={cx(
+        getTabsClass('__content', scoped),
+        isActive ? 'slds-show' : 'slds-hide'
+      )}
+      id={id}
+      role="tabpanel"
+    >
+      {isActive === null ? null : children}
```

Comment:
This is a somewhat confusing double-use of `isActive`. I'd make this two props: `isActive` for controlling the style, `renderWhenInactive` to control the hidden rendering behaviour.

Reply:
I tried to minimize the amount of internally used props in the user-facing API Tab `propTypes`. I figured that this is ok to do since I have full control over these props here and added specs for it.